apps/fluent-tester/src/E2E/MenuButton/pages/MenuButtonPageObject.win.ts

```diff
@@ -1,80 +1,80 @@
-import {
-  MENU_BUTTON_TESTPAGE,
-  MENU_BUTTON_TEST_COMPONENT,
-  HOMEPAGE_MENUBUTTON_BUTTON,
-  MENU_BUTTON_NO_A11Y_LABEL_COMPONENT,
-  MENU_ITEM_1_COMPONENT,
-} from '../../../TestComponents/MenuButton/consts';
-import { BasePage, By } from '../../common/BasePage.win';
+// import {
```

Comment:
Not sure if you've already answered this but why are the MenuButton E2E test pages commented out? Should they be?

Reply:
Same as below, ran into local issue but I figured it out, adding it back now.